### PR TITLE
`Exam Mode`: Show exam announcements only for started exams

### PR DESCRIPTION
**Pull request by coolchock**
#### General

- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

In issues #9010 and #9021, two problems regarding exam announcements were reported. Both issues involved exam announcements being displayed before the exam started and after the exam ended when the user opened the Exams tab on the course page. This pull request resolves both #9010 and #9021.

### Description

The jhi-exam-live-events-button was present in the exam-participation-cover.component.html, which caused announcements to be displayed on the exam cover page. I removed the jhi-exam-live-events-button, so now announcements are only displayed if the user has started the exam and opened the exam-participation.component.html.

#### Exam Mode Testing

Prerequisites:
- 1 Instructor
- 1 Student
- 1 Exam

1. Log in to Artemis as an instructor in one browser and as a student in another browser/tab.
2. As the student, go to your course -> Exams.
3. As the instructor, go to Course Management -> Exams and select your exam.
4. Change the working time of the exam, create an announcement, and change the problem statement of an exercise. Verify all three cases. Ensure that at least one case is tested when the exam is visible to the student, but the working time has not yet started.
5. As the student, verify that no announcements are displayed before the exam has started.
6. Start the exam and verify that announcements are displayed.

[](https://artemis-test1.artemis.cit.tum.de)
[](https://artemis-test2.artemis.cit.tum.de)
[](https://artemis-test3.artemis.cit.tum.de)
[](https://artemis-test4.artemis.cit.tum.de)
[](https://artemis-test5.artemis.cit.tum.de)
[](https://artemis-test6.artemis.cit.tum.de)
[](https://artemis-test9.artemis.cit.tum.de)

#### Performance Review
- [ ] I (as a reviewer) confirm that the client changes (in particular related to REST calls and UI responsiveness) are implemented with a very good performance 
- [ ] I (as a reviewer) confirm that the server changes (in particular related to database calls) are implemented with a very good performance
#### Code Review
- [ ] Code Review 1
- [ ] Code Review 2
#### Manual Tests
- [ ] Test 1
- [ ] Test 2
#### Exam Mode Test
- [ ] Test 1
- [ ] Test 2

## Summary by CodeRabbit

- **New Features**
	- Simplified user interface by removing the live events button during exam wait times and submission issues.
	- Focus shifted towards presenting exam title and general information, enhancing usability and clarity for students.

### CODE REVIEW

**Review by Strohgelaender — DISMISSED** [review]

**Review by edkaya — COMMENTED** [review]
Tested on ts4, did not get any notifications before I start the exam 👍 However, I think we still want to display notifications in end page, because it is part of an active exam period. Removing from that notification button also breaks the end page layout. (However, it is not so important, since we have a new end page design.)

Screenshot 2024-07-26 at 13 37 23

File: src/main/webapp/app/exam/participate/exam-cover/exam-participation-cover.component.html [hunk 1]
```diff
@@ -1,8 +1,5 @@
 @if (!startView && !studentFailedToSubmit) {
     <div class="w-100 d-flex">
-        <div class="col-md-3">
```
Comment (edkaya) [inline src/main/webapp/app/exam/participate/exam-cover/exam-participation-cover.component.html:3]: Here should be added again

File: src/main/webapp/app/exam/participate/exam-cover/exam-participation-cover.component.html [hunk 2]
```diff
@@ -1,8 +1,5 @@
...
-            <jhi-exam-live-events-button />
-        </div>
```
Comment (edkaya) [inline src/main/webapp/app/exam/participate/exam-cover/exam-participation-cover.component.html:5]: Here should be added again (Line 3-5)

**Review by edkaya — COMMENTED** [review]
Besides, I also realized that if there are some changes before an exam starts, the notifications are not displayed in the welcome page but still displayed directly after we start with the exam, which I am not sure if it is the intended behaviour. Because the changes before the exam's start date should be irrelevant for the students in my opinion. 
Screenshot 2024-07-26 at 18 05 30

**Review by JohannesStoehr — CHANGES_REQUESTED** [review]
This PR should also adapt the server so it also makes sure that no information is sent to the students that is not supposed to be sent

**Comment by edkaya** [comment]
> This PR should also adapt the server so it also makes sure that no information is sent to the students that is not supposed to be sent

Server part will be adapted here #9136

**Review by SimonEntholzer — DISMISSED** [review]
code

**Pull request closed without merging**